src/Stratis.Bitcoin.Features.Dashboard/Dashboard.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.DependencyInjection;
+using Stratis.Bitcoin.Builder;
+using Stratis.Bitcoin.Builder.Feature;
+using Stratis.Bitcoin.Features.Api;
+using Stratis.Bitcoin.Features.Dashboard.Controllers; 
+
+namespace Stratis.Bitcoin.Features.Dashboard
+{
+    /// <summary>
+    /// Dashboard Feature
+    /// </summary>
+	public class Dashboard : FullNodeFeature 
```

Comment:
`DashboardFeature`

Reply:
Please also rename the file to `DashboardFeature.cs`